packages/nodes-base/credentials/LoneScaleApi.credentials.ts

```diff
@@ -0,0 +1,38 @@
+import type {
+	IAuthenticateGeneric,
+	ICredentialTestRequest,
+	ICredentialType,
+	INodeProperties,
+} from 'n8n-workflow';
+
+export class LoneScaleApi implements ICredentialType {
+	name = 'loneScaleApi';
+
+	displayName = 'LoneScale API';
+
+	properties: INodeProperties[] = [
+		{
+			displayName: 'API Key',
+			name: 'apiKey',
+			type: 'string',
+			typeOptions: { password: true },
+			default: '',
+		},
+	];
+
+	authenticate: IAuthenticateGeneric = {
+		type: 'generic',
+		properties: {
+			headers: {
+				Authorization: '=Bearer {{$credentials.apiKey}}',
```

Comment:
This header key should be...
```
'X-API-KEY': '={{$credentials.apiKey}}',
```
Using the Authorization header with Bearer leads to an authentication failure